prow/config.yaml

```diff
@@ -37,6 +36,7 @@ deck:
       - "buildlog"
       "artifacts/junit.*\\.xml":
       - "junit"
+    announcement: "The old job viewer, Gubernator, has been deprecated in favour of this page, Spyglass.{{if .ArtifactPath}}For now, the old page is <a href='https://gubernator.k8s.io/build/{{.ArtifactPath}}'>still available</a>.{{end}} Please send feedback to sig-testing."
```

Comment:
Steve you'll need to set this config value to have it link back to your gubernator